flytekit/clis/sdk_in_container/package.py

```diff
@@ -54,17 +54,18 @@
     is_flag=True,
     default=False,
     required=False,
-    help="[Will be deprecated, see --copy] This flag enables fast packaging, that allows `no container build`"
-    " deploys of flyte workflows and tasks. You can specify --copy all/auto instead"
+    help="[Deprecated, see --copy] This flag enables fast packaging, that allows `no container build`"
+    " deploys of flyte workflows and tasks. You should specify --copy all/auto instead"
     " Note this needs additional configuration, refer to the docs.",
 )
 @click.option(
     "--copy",
     required=False,
     type=click.Choice(["all", "auto", "none"], case_sensitive=False),
-    default=None,  # this will be changed to "none" after removing fast option
+    default="none",
```

Comment:
As long as `--fast` is in the codebase, someone can still run `--fast --copy auto`, which is conflicting/confusing. I like keeping the default for `copy` as `None` and raising the current error.

When we actually remove `--fast` as an option, then I feel safe enough to set `default="none"`.

Reply:
we won't remove it though for a while, at least a year or two... so just keep it around?

Reply:
can we detect (and fail) in this case? Also, add a message that explains how to fix the issue.

Reply:
also did you mean this for `package` specifically or the commands in general?  if so we can just discard this PR... but that leaves the old logic and doesn't exercise the new logic unless it's opted into.

Reply:
we can detect and fail yeah... but i'd like to exercise the new code